website/layouts/index.html

```diff
@@ -52,7 +52,33 @@ <h4 class="font-weight-bold">Downsampling & Compaction</h4>
         </div>
     </div>
 </div>
-<div class="container-fluid">
+<div class="container py-5">
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Created by</h3></div>
+    </div>
+    <div class="row">
+        <div class="col-4 col-xl-2 offset-4 offset-xl-5 py-3">
+            <div class="img-sponsor">
+                <a href="https://improbable.io" rel="nofollow" target="_blank"><img src="{{ "/logos/improbable.png" | relURL }}" class="img-fluid" alt="Improbable"></a>
+            </div>
+        </div>
+    </div>
+    {{ if $.Site.Data.sponsors.sponsors }}
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Used by</h3></div>
+    </div>
+    <div class="row">
```

Comment:
Wow you automated this <3 

Can we add link to `how-to-contribute-to-docs.md#Logos` with message something like "Want to be added to list? Follow this"

Reply:
How about 
![image](https://user-images.githubusercontent.com/781688/56792777-a47b0c80-680a-11e9-81e7-555583623346.png) ?

Reply:
Yes! We can drop this when we have lot's of those ? And let's do it just very grey almost transparent?